addons/tmf_loadouts/loadouts/ru_army_template_2020.hpp

```diff
@@ -617,7 +619,7 @@ class jp : baseMan {
 	goggles[] = {"default"};
 	headgear[] = {"CUP_H_RUS_ZSH_Shield_Down"};
 	uniform[] = {"CUP_U_B_USMC_PilotOverall"};
-	vest[] = {"CUP_V_RUS_6B45_1"};
+	vest[] = {"Aircrew_vest_2_NH"};
```

Comment:
Not sure about this.

Reply:
Reverting changes, on second look I agree.